### PR TITLE
Transaction Logs - Backend Changes

### PR DESCRIPTION
**Pull request by negue**
Extracted Backend Changes from #14018

### CODE REVIEW

File: website/common/locales/en/settings.json [hunk 1]
```diff
@@ -204,11 +204,13 @@
     "transaction_gift_send": "Gifted to",
     "transaction_gift_receive": "Received from",
     "transaction_create_challenge": "Created challenge",
+    "transaction_create_bank_challenge": "Created bank challenge",
```
Comment (SabreCat) [inline website/common/locales/en/settings.json:207]: I've never heard this term "bank challenge"--based on the rest of the code, it looks like that's when a Challenge is created that draws Gems from Guild balance?

**Review by SabreCat — COMMENTED** [review]
Got one question, terminology that might need team input, but overall I think this piece is pretty simple!

File: website/common/locales/en/settings.json (hunk 1 above)
Comment (negue) [reply website/common/locales/en/settings.json:207]: yes thats a challenge that is "payed" by the guild'S balance

@Tressley maybe a different sentence is needed or some kind of help label ?

Comment (SabreCat) [reply website/common/locales/en/settings.json:207]: With the filtering so this only shows up for user support folks, we're free to adjust the language at will. ok as is!

**Pull request merged**